test/configCases/entry/single-entry-point/webpack.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	entry: "./single-entry-point",
+	output: {
```

Comment:
For config test cases, behind the scenes if you look in `ConfigTestCases.test.js`:

```javascript
					optionsArr.forEach(function(options, idx) {
						if(!options.context) options.context = testDirectory;
						if(!options.entry) options.entry = "./index.js";
						if(!options.target) options.target = "async-node";
						if(!options.output) options.output = {};
						if(!options.output.path) options.output.path = outputDirectory;
						if(typeof options.output.pathinfo === "undefined") options.output.pathinfo = true;
						if(!options.output.filename) options.output.filename = "bundle" + idx + ".js";
						if(!options.output.chunkFilename) options.output.chunkFilename = "[id].bundle" + idx + ".js";
					});
```

The output path is already configure. This is what is failing the test I believe because it is needing an output path. My recommendation would be to leave out the output property since it has a safe and sane default thanks to the test cases.